securedrop/source_templates/howto-disable-js.html

```diff
@@ -3,7 +3,7 @@
 
 <h1>Disable JavaScript to Protect Your Anonymity</h1>
 
-<p>JavaScript is a widely used programming language for creating interactive web pages. Unfortunately, JavaScript is also the most common source of security vulnerabilities in browsers. In August 2013, the FBI wrote malware that used a JavaScript exploit to de-anonymize Tor users (http://www.wired.com/threatlevel/2013/09/freedom-hosting-fbi). JavaScript exploits are also mentioned in documents describing the NSA's Egotistical Giraffe (http://www.theguardian.com/world/interactive/2013/oct/04/egotistical-giraffe-nsa-tor-document) program, which also has the goal of targeting and de-anonymizing Tor users.</p>
+<p>JavaScript is a widely used programming language for creating interactive web pages. Unfortunately, JavaScript is also the most common source of security vulnerabilities in browsers. In August 2013, the FBI wrote malware that used a JavaScript exploit to <a href='http://www.wired.com/threatlevel/2013/09/freedom-hosting-fbi'>de-anonymize Tor users</a>. JavaScript exploits are also mentioned in documents describing the NSA's <a href='http://www.theguardian.com/world/interactive/2013/oct/04/egotistical-giraffe-nsa-tor-document'>Egotistical Giraffe</a> program, which also has the goal of targeting and de-anonymizing Tor users.</p>
```

Comment:
Can you make the Wired link HTTPS since it supports it? HTTPS Everywhere might do this for us, but it doesn't hurt.